server/devpi_server/extpypi.py

```diff
@@ -181,6 +181,19 @@ def del_project(self, project):
             self.cache_retrieve_times.expire(project)
             self.key_projects.set(projects)
 
+    def del_versiondata(self, project, version, cleanup=True):
+        project = normalize_name(project)
+        if not self.has_project_perstage(project):
+            raise self.NotFound("project %r not found on stage %r" %
+                                (project, self.name))
+        # since this is a mirror, we only delete the actual files
```

Comment:
Better comment:
As this is a mirror, there is no metadata. Hence, only delete mirrored files.